src/Umbraco.Infrastructure/Umbraco.Infrastructure.csproj

```diff
@@ -45,38 +40,11 @@
     <PackageReference Include="SixLabors.ImageSharp" Version="2.1.3" />
     <PackageReference Include="System.IO.FileSystem.AccessControl" Version="5.0.0" />
     <PackageReference Include="System.Security.Cryptography.Pkcs" Version="6.0.1" />
-    <PackageReference Include="System.Text.Encodings.Web" Version="6.0.0" /> <!-- Explicit updated this nested dependency due to this https://github.com/dotnet/announcements/issues/178-->
```

Comment:
The issue is still open, so is this not still needed?

Reply:
The CVE is fixed in version 5.0.1 (see https://github.com/advisories/GHSA-ghhp-997w-qr28) and we're already using 6.0.0. You can check this by running `dotnet list package --include-transitive`.

Reply:
Ahh yes makes sense 😄